app/controllers/activities_controller.rb

```diff
@@ -7,6 +7,7 @@ class ActivitiesController < ApplicationController
 
   def index
     @team = Team.find_by id: params[:team_id] if params[:team_id]
+    flash.now[:notice] = "You can add comments now! Click on the title of a post and see what emerges"
```

Comment:
Does an 'Add comment' link on the index page even work? A comment always references a single record so we can only really offer that option for a show action.

I would remove the flash. It's good to advertise a new feature but the flash system is not the right channel. I've announced the last major changes via twitter; @anasofiapinho retweeted it and I hope that at least a few students interested in the Teams App caught the news. If not – they'll learn about it by some other means eventually 😄